general/press-and-publications.rst

```diff
@@ -0,0 +1,111 @@
+Press and Publications
+======================
+
+In this page, we aim to provide you links to presentations and popular
+blog posts about OpenWISP.
+
+Presentations:
+--------------
+
+PyCon Italia 2017
+^^^^^^^^^^^^^^^^^
+
+- video: https://www.youtube.com/watch?v=tm7Opg3QyZk
+- slides: `<https://www.pycon.it/conference/talks/applying-the-unix-
+  philosophy-to-django-projects-a-report-from-the-real-world>`_
+
+OpenWRT Summit 2017
+^^^^^^^^^^^^^^^^^^^
+
+- video: https://www.youtube.com/watch?v=n531yTtJimU
+- slides: http://static.nemesisdesign.net/openwisp2-openwrt-summit-2017/
+
+FOSDEM 2017
+^^^^^^^^^^^
+
+- abstract: https://archive.fosdem.org/2017/schedule/event/openwisp2
+
+IETF Meeting 103 Bangkok
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+- `<https://datatracker.ietf.org/meeting/103/materials/slides-103-gaia-
+  openwisp-a-hackable-network-management-system-for-the-21st-centry-00>`_
+
+django-freeradius by Fiorella De Luca at PyCon Italy:
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+- video: https://www.youtube.com/watch?v=Yapdso_6EGA
+- abstract: https://www.pycon.it/conference/talks/django-freeradius
+
+Opening Proprietary Networks with OpenWISP
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+*lightning talk at DjangoCon Europe 2017*:
```

Comment:
leave blank line after the heading, add a link to the event, apply to similar instances,

Reply:
I didn't leave a line after that heading because it was continuing on the next line. I researched but couldn't find a way to do 'multi-line' Headings as I wanted to put all of that on a single line. I will add the links tho...